docs/source/pages/plotting.rst

```diff
@@ -1,3 +1,5 @@
+.. _plotting:
```

Comment:
isn't it a bit confusing to have the page and label the same name?

Reply:
maybe, but we have the same thing multiple places in the docs